WinUIGallery/SettingsPage.xaml

```diff
@@ -111,11 +111,19 @@
                         IsTextSelectionEnabled="True"
                         Text="{x:Bind Version}" />
                     <toolkit:SettingsExpander.Items>
-                        <toolkit:SettingsCard Header="To clone this repository">
+                        <toolkit:SettingsCard
+                            x:Name="toCloneRepoCard"
+                            Click="toCloneRepoCard_Click"
+                            Header="To clone this repository"
+                            IsClickEnabled="True">
                             <TextBlock
+                                x:Name="gitCloneTextBlock"
```

Comment:
I think we should probably use a monospaced font and a separate background for this text (though this text was already like this before this PR)

Reply:
I changed the font to `Consolas` just like the other places (Iconography/Geometry) that has a copy button.

![image](https://github.com/user-attachments/assets/ff372159-260d-440f-8b64-23cc5bbe8d21)

Reply:
> a separate background for this text

I'm not sure what kind of background you mean. 🤔
Is there any section in the app that I can use as a reference?

Reply:
Huh interesting, I thought that we had a specific background for source code but apparently we no longer have that.
Thank you for using Consolas btw!